diff -Nur -x '*.orig' -x '*~' xpother//lib/X11/Protocol/Ext/DAMAGE.pm xpother.new//lib/X11/Protocol/Ext/DAMAGE.pm
--- xpother//lib/X11/Protocol/Ext/DAMAGE.pm	2011-04-19 18:14:01.000000000 +1000
+++ xpother.new//lib/X11/Protocol/Ext/DAMAGE.pm	2011-04-19 18:15:23.000000000 +1000
@@ -252,7 +252,7 @@
 region from the damage object.  This fetch is atomic, so nothing is lost if
 the client is a bit lagged etc.
 
-See F<examples/damage-duplicate.pl> in the C<X11-Protocol-Other> sources for
+See F</usr/share/doc/libx11-protocol-other-perl/examples/examples/damage-duplicate.pl.gz> for
 one way to use damage to duplicate a window in real-time.
 
 =head1 REQUESTS
diff -Nur -x '*.orig' -x '*~' xpother//lib/X11/Protocol/Ext/X_Resource.pm xpother.new//lib/X11/Protocol/Ext/X_Resource.pm
--- xpother//lib/X11/Protocol/Ext/X_Resource.pm	2011-04-19 18:02:30.000000000 +1000
+++ xpother.new//lib/X11/Protocol/Ext/X_Resource.pm	2011-04-19 18:14:18.000000000 +1000
@@ -194,7 +194,7 @@
 resource database of widget settings and user preferences (per
 L<X(7)/RESOURCES>).
 
-See F<examples/xresource-print.pl> in the C<X11-Protocol-Other> sources for
+See F</usr/share/doc/libx11-protocol-other-perl/examples/xresource-print.pl> for
 a simple dump of the resources reported.
 
 =head1 REQUESTS
@@ -278,8 +278,7 @@
     print "using $windows many windows, and $grabs passive grabs";
 
 C<List::Pairwise> has a C<mapp()> and other things to work with this sort of
-two-at-a-time list too (see F<examples/xresource-pairwise.pl> in the
-C<X11-Protocol-Other> sources).
+two-at-a-time list too (see F</usr/share/doc/libx11-protocol-other-perl/examples/xresource-pairwise.pl>).
 
 Generally a count entry is only present when the client has 1 or more of the
 thing.  So if no pixmaps then no C<PIXMAP> entry at all.
